--- src\dynamic_tab.rs	Sun Jul 23 18:21:28 2006
+++ src\dynamic_tab.rs	Thu Jan 19 07:22:06 2023
@@ -6,1 +6,3 @@
-
+
+use crate::{TabIndex, NodeIndex};
+
@@ -217,1 +217,8 @@
-    fn context_menu(&mut self, _ui: &mut Ui, _tab: &mut Self::Tab) {}
+    fn context_menu(
+        &mut self,
+        _ui: &mut Ui,
+        _tab: &mut Self::Tab,
+        _tabindex: TabIndex,
+        _nodeindex: NodeIndex,
+    ) {
+    }
--- src/lib.rs	Sun Jul 23 18:21:28 2006
+++ src/lib.rs	Thu Jan 19 07:22:06 2023
@@ -161,1 +161,8 @@
-    fn context_menu(&mut self, _ui: &mut Ui, _tab: &mut Self::Tab) {}
+    fn context_menu(
+        &mut self,
+        _ui: &mut Ui,
+        _tab: &mut Self::Tab,
+        _tabindex: TabIndex,
+        _nodeindex: NodeIndex,
+    ) {
+    }
@@ -451,1 +451,1 @@
-                                        tab_viewer.context_menu(ui, tab);
+                                        tab_viewer.context_menu(ui, tab, tab_index, node_index);
